app/assets/javascripts/messages.js

```diff
@@ -13,16 +20,8 @@ $(document).ready(function(){
     fillLikeOption(this);
   });
 
-  $('like-options like-option').on('keyup', function(e) {
```

Comment:
This must be present so that tabbed-to like-options can be selected with the enter key